scripts/bench_gigatoken.py

```
@@ -0,0 +1,132 @@
+    python scripts/bench_gigatoken.py --model gpt2 --corpus data.txt --wandb-project gigatoken-bench
+"""
+
+import argparse
+import time
+
+from transformers import AutoTokenizer
+
+
+def load_texts(path, limit):
+    with open(path, encoding="utf-8") as f:
+        texts = [line for line in (l.strip() for l in f) if line]
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)